Alloy/lib/alloy/sync/properties.js

```diff
@@ -1,6 +1,7 @@
 var Alloy = require('/alloy'),
 	_ = require('/alloy/underscore')._,
-	TAP = Ti.App.Properties;
+	TAP = Ti.App.Properties,
+  Backbone = Alloy.Backbone;
```

Comment:
<!--
  1 failure:  Alloy/lib/alloy/s...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Alloy/lib/alloy/sync/properties.js;
  Line: 4;
-->
- :no_entry_sign: Alloy/lib/alloy/sync/properties.js line 4 – Expected indentation of 1 tab but found 2 spaces. (indent)